nat-lab/tests/test_tcli.py

```diff
@@ -1,95 +0,0 @@
-import asyncio
```

Comment:
Is there another simple test that only checks DERP connection? I found this test very useful to check if basic functionality is working.

Reply:
any test from `test_derp_connect.py` and also `test_telio_tasks_with_all_features` will be doing the same thing, when I implement usage of `setup_mesh_nodes`